houndigrade/cli.py

```diff
@@ -0,0 +1,227 @@
+"""Command line script for inspecting attached volumes."""
+import glob
+import json
+import os
+import subprocess
+from contextlib import contextmanager
+from gettext import gettext as _
+
+import click
+from kombu import Connection, Exchange, Queue
+
+INSPECT_PATH = '/mnt/inspect'
+
+
+@click.command()
+@click.option('--cloud',
+              '-c',
+              default='aws',
+              help='Cloud in which we are performing the inspection.',
+              type=click.Choice(['aws', 'gcp', 'azure']))
+@click.option('--target',
+              '-t',
+              multiple=True,
+              type=(str, click.Path(exists=True)),
+              required=True,
+              help=_('Inspection target, the cloud specific image identifier'
+                     ' and path to the attached drive on the machine. e.g.'
+                     ' -t ami-12312839312 /dev/sda'))
+@click.option('--debug',
+              is_flag=True,
+              help=_('Print debug output.'))
+def main(cloud, target, debug):
+    """
+    Mounts provided volumes and inspects them.
+
+    The script takes provided target information and checks if there are any
+    partitions on that device, then it loops through, mounting every partition
+    and inspecting it. As it loops through it builds a dictionary of results
+    that gets placed on a queue once the processing is done.
+
+    """
+    click.echo(_('Provided cloud: {}'.format(cloud)))
```

Comment:
The matryoshka doll is closing incorrectly order here. You want to `.format` to happen *after* the translation occurs. Like this:

```
_('Provided cloud: {}').format(cloud)
```